cmd/ztest/ztest.c

```diff
@@ -7384,7 +7384,7 @@ main(int argc, char **argv)
 	 * segments can be reconstructed in a reasonable amount of time
 	 * when reconstruction is known to be possible.
 	 */
-	zfs_reconstruct_indirect_damage_fraction = 4;
+	zfs_reconstruct_indirect_damage_fraction = 100;
```

Comment:
can you explain this a bit more?  why can't we do vdev_indirect_splits_damage() every time?  Why is 1/100th of the time OK?

Reply:
This change was from @behlendorf. We were seeing several cases where reconstruction was taking a very long time. In fact, with 16 segments we were seeing ztest deadman timeouts. We wanted to reduce this workload so ztest has time to test other things so we lowered the max damaged segments to 12 and decreased how often they would occur.

Reply:
I see, so this isn't an issue of correctness (too much damage causing it to be unrecoverable), just performance (too much damage causing us to spend too much time doing successful recovery).

Reply:
Yes, exactly.